test/Transfer/Adapter/HttpTestMockAdapter.php

```diff
@@ -50,7 +52,7 @@ public static function uPTest($id)
     }
 
     public function switchApcToUP()
-    {
+    {	static::$aa = false;
```

Comment:
I've pulled the changes in locally, and none of the changes to this particular class are necessary; tests pass with and without them.